### ISSUE TITLE
Burn Book: Missing recorder examples

### ISSUE THREAD

**Issue by louisfd** [tracel-ai/burn#1156]
We should add more comprehensive examples and documentation on how to load and save weights.

**Comment by antimora** [comment]
I agree
